akka-projection-grpc/src/main/protobuf/akka/projection/grpc/event_consumer.proto

```diff
@@ -59,6 +59,7 @@ message ConsumerEventInit {
 
 message ConsumerEventStart {
   repeated FilterCriteria filter = 1;
+  ReplicaInfo replica_info = 6;
```

Comment:
added also to ConsumerEventStart but setting it and implementing corresponding filter on the producer side for RES must be done in @johanandren 's #1073 (or after that)

Reply:
I'll do that in that PR once rebased on top of this after merged